packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -73,8 +73,8 @@ const CardSection = ({ title, cards, theme }) => {
                     <Card
                       key={index}
                       image={card.image}
-                      title={card.title}
-                      content={card.copy}
+                      title={card.eyebrow ? card.eyebrow : card.title}
```

Comment:
yeah but `the card pattern does not have eyebrow prop yet` that is the reason I added.
![image](https://user-images.githubusercontent.com/53789187/74541859-6f15eb80-4f10-11ea-998e-ef9eea9a5508.png)

Reply:
That's fine. I'd still put it where it's supposed to go. You can form it based on Anna's pull request. The eyebrow is not he same thing as the title here, and it's not an either or because they are two different things and even have to exist at the same time.

Reply:
Okay